allensdk/test/brain_observatory/behavior/test_behavior_ophys_lims_api.py

```diff
@@ -260,51 +270,47 @@ def test_corrected_fluorescence_trace_order(monkeypatch, tmpdir):
     can reorder ROIs to align with what is in the cell_specimen_table
     """
 
-    out_fname = os.path.join(tmpdir, 'dummy_ftrace_data.h5')
+    out_fname = os.path.join(tmpdir, "dummy_ftrace_data.h5")
```

Comment:
I was thinking that `tmpdir` and `tmp_path` were equivalent, but, I think we should start moving to using `tmp_path` as the underlying `py.path.local` is in maintenance mode. Probably `pytest` will make the switch for us at some point.
In either case, no need for `os` here, could just be `tmpdir / "filename.h5"`
no change requested.